.gitignore

```diff
@@ -26,6 +26,6 @@ package-lock.json
 coverage
 cypress
 dist
-public
+/public
```

Comment:
Will this ignore `public` from examples now?

Reply:
I believe those are all `dist` but if they did create such a folder it would not be ignored

The goal here is to ignore `./public` but not `.storybook/public`